### ISSUE TITLE
(Changing the) Location of the tutorials directory?

### ISSUE THREAD

**Issue by jypeter** [UV-CDAT/uvcdat#451]
I have found by chance the (new) location of the _tutorials_ directory!

```
install/bin/tutorials
```

Is there a good reason to bury it down there? Would it be possible to have the _tutorials_ directory just above, at the **same** level as the sample data?

```
install/sample_data
```

The location of these directories in the distribution should probably be mentioned on the following web pages:
- http://uv-cdat.llnl.gov/tutorials.html
- https://github.com/UV-CDAT/uvcdat/wiki/Building-UVCDAT

It would also be **very useful** to document on a web page the _trick_ you use to locate the sample data directory

```
sys.prefix + '/sample_data'
```

I know that such a trick exists, but I keep forgetting it. And it would be useful if I used it more often in my examples and test cases

**Comment by doutriaux1** [comment]
Oh! These are nice! I forgot about these! @chaosphere2112 @mattben you can probably use them on the gallery and other web-pages let's try to put this on the website for 2.2

**Comment by chaosphere2112** [comment]
I'll go ahead and work on that next week. I'll assign this to myself.

**Comment by doutriaux1** [comment]
@chaosphere2112 progress on this?

**Comment by aashish24** [comment]
@chaosphere2112 this is on you :+1:

**Comment by chaosphere2112** [comment]
@doutriaux1 @aashish24 Nope, haven't done anything with these yet. "Next Week" was supposed to be "after the release" :wink:

**Comment by doutriaux1** [comment]
in that case moving to 2.3

**Comment by aashish24** [comment]
Shuould we move it to 3.0? @doutriaux1 ?

**Comment by chaosphere2112** [comment]
Can we just update these to go to /share?

**Comment by durack1** [comment]
@chaosphere2112 `milestone 3.0`?
